### PR TITLE
doc(system restore): support backup backing image in system backup restore

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: https://github.com/longhorn/longhorn/issues/5085

support backup backing image in system backup restore

### CODE REVIEW

File: content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ It includes below resources associating with the Longhorn system:
 - ServiceAccounts
 - StorageClasses
 - Volumes
-
-> **Warning:** Longhorn does not backup `BackingImages`. We will improve this part in the future. See [Restore Longhorn System - Prerequisite](../restore-longhorn-system/#prerequisite) for restoring volumes created with the backing image.
+- BackingImages
```
Comment (c3y1huang) [inline content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md:39]: This list is sorted alphabetically. Please move it up.

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md (hunk 1 above)
Comment (ChanYiLin) [reply content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md:39]: ok got it

Comment (ChanYiLin) [reply content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md]: Fixed.

**Review by c3y1huang — COMMENTED** [review]
Could you check if [this prerequisite statement](https://github.com/longhorn/website/blob/master/content/docs/1.7.0/advanced-resources/system-backup-restore/restore-longhorn-system.md?plain=1#L44-L46) is still valid?

**Review by c3y1huang — APPROVED** [review]

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**